### PR TITLE
Update get_email to prevent error on empty body message

### PR DESCRIPTION
**Pull request by tboulogne**
Pull request Regarding #622
Avoid the error when message body is empty

### CODE REVIEW

File: helpdesk/management/commands/get_email.py [hunk 1]
```diff
@@ -400,8 +400,11 @@ def ticket_from_message(message, queue, logger):
         mail = BeautifulSoup(part.get_payload(), "lxml")
         if ">" in mail.text:
             body = mail.find('body')
-            body = body.text
-            body = body.encode('ascii', errors='ignore')
+            try:
+               body = body.text
+               body = body.encode('ascii', errors='ignore')
+            except AttributeError:
+               body = ''
```
Comment (timthelion) [inline helpdesk/management/commands/get_email.py]: I think it would be better to default to `body = mail.text`.

Comment (tboulogne) [reply helpdesk/management/commands/get_email.py]: In a first time i think like you and make `body = mail.text` but i got html code into the ticket.
so i set body to `''`.

Comment (timthelion) [reply helpdesk/management/commands/get_email.py]: It seems to me like that's better than showing nothing at all.

Comment (gwasser) [reply helpdesk/management/commands/get_email.py]: We could maybe use BeautifulSoup such as `mail.get_text()` to try to filter out any html present.

**Pull request closed without merging**